.github/workflows/publish-images.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and publish images
+
+on:
+  workflow_dispatch:
+
+  push:
```

Comment:
I think it'd be better to do it on release/tag rather than on push, as discussed earlier